test/components/views/avatars/DecoratedRoomAvatar-test.tsx

```diff
@@ -39,14 +47,14 @@ describe("DecoratedRoomAvatar", () => {
         room = new Room(ROOM_ID, mockClient, mockClient.getUserId() ?? "", {
             pendingEventOrdering: PendingEventOrdering.Detached,
         });
+    });
 
+    it("shows an avatar with globe icon and tooltip for public room", async () => {
         const dmRoomMap = {
             getUserIdForRoomId: jest.fn(),
         } as unknown as DMRoomMap;
         jest.spyOn(DMRoomMap, "shared").mockReturnValue(dmRoomMap);
```

Comment:
we should really clear these mocks/spies after the test completes, otherwise it is going to be confusing if people add more tests and the mock implementations leak into the new test.

Could you add an `afterEach` function with `jest.restoreAllMocks()` ?